### DISCUSSION TITLE
several tests failed when installing on HPC cluster

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#26966] [Q&A General]
I am installing moose on a hpc cluster. The compiling is ok but when I run tests, it reports the following error:
![tests](https://github.com/idaholab/moose/assets/43910101/ad33e099-2b4b-4c64-a524-b5c64f77d10a)
How can I detect what is wrong and fix it?

**Comment by GiudGiud** [comment]
Hello

The skips are fine. 
These diffs are due to a different version of exodus iirc and are not problematic.
To make sure, instead of pasting the summary, you can paste the log of the failed tests. 

./run_tests --failed-tests will get you that

Guillaume

**Reply by RWTHLHK** [reply]
ok thanks!
